### PR TITLE
Model check

### PR DESCRIPTION
**Pull request by conig**
Depends on #49

### CODE REVIEW

**Comment by conig** [comment]
Closes #51

File: doc/multiverse.pdf
Comment (tarensanders) [inline doc/multiverse.pdf:1]: There's some pretty wild values of kurtosis for PA Volume. Do we need to check that out?

**Review by tarensanders — APPROVED** [review]

File: doc/multiverse.pdf
Comment (conig) [reply doc/multiverse.pdf:1]: I'll log an issue. Let's chat to Phil before we do anything drastic. I'm pretty sure he won't be worried. At least skew looks good across the board.

Comment (conig) [reply doc/multiverse.pdf:1]: lol OK those are very crazy kurtosis values... Will log the issue and look into it.

**Pull request merged**